django_cas_ng/__init__.py

```diff
@@ -15,6 +15,7 @@
     'CAS_RENEW': False,
     'CAS_IGNORE_REFERER': False,
     'CAS_LOGOUT_COMPLETELY': True,
+    'CAS_FORCE_CHANGE_USERNAME_CASE': False,
```

Comment:
default value should be None. if use 'False' it is indicate this is a boolean type.